apps/web/pages/getting-started/[[...step]].tsx

```diff
@@ -178,6 +178,10 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
     throw new Error("User from session not found");
   }
 
+  if (user.completedOnboarding) {
```

Comment:
redirect if the user is onboarded to prevent incorrect value in selected timezone.